airbyte-integrations/connectors/source-notion/integration_tests/expected_records.txt

```diff
@@ -1,37 +1,23 @@
-{"stream": "blocks", "data": {"object": "block", "id": "db8c1f18-28f9-4230-8af3-9377d2e52aaf", "created_time": "2021-10-16T10:06:00.000Z", "last_edited_time": "2021-10-16T10:06:00.000Z", "has_children": false, "archived": false, "type": "paragraph", "paragraph": {"text": [{"type": "text", "text": {"content": "slaksjdfl", "link": null}, "annotations": {"bold": false, "italic": false, "strikethrough": false, "underline": false, "code": false, "color": "default"}, "plain_text": "slaksjdfl", "href": null}]}}, "emitted_at": 1634793435000}
-{"stream": "blocks", "data": {"object": "block", "id": "effd0166-724d-4a1a-8e41-87ee4aaf2499", "created_time": "2021-10-16T10:06:00.000Z", "last_edited_time": "2021-10-16T10:06:00.000Z", "has_children": false, "archived": false, "type": "paragraph", "paragraph": {"text": []}}, "emitted_at": 1634793435000}
-{"stream": "blocks", "data": {"object": "block", "id": "5c8a1e06-5050-458d-9e5a-bc47d4a8c5f4", "created_time": "2021-10-16T10:06:00.000Z", "last_edited_time": "2021-10-16T10:06:00.000Z", "has_children": true, "archived": false, "type": "toggle", "toggle": {"text": [{"type": "text", "text": {"content": "sdfsadf", "link": null}, "annotations": {"bold": false, "italic": false, "strikethrough": false, "underline": false, "code": false, "color": "default"}, "plain_text": "sdfsadf", "href": null}]}}, "emitted_at": 1634793435000}
+{"stream": "users", "data": {"object": "user", "id": "f5ac1fcb-a06b-4dcc-80e5-403c40dfb38a", "name": "Airyte", "avatar_url": null, "type": "person", "person": {"email": "integration-test@airbyte.io"}}, "emitted_at": 1671456048322}
+{"stream": "users", "data": {"object": "user", "id": "6f1be537-9a62-4686-bb6a-02918894529f", "name": "karina.kuznietsova@zazmic.com", "avatar_url": null, "type": "person", "person": {"email": "karina.kuznietsova@zazmic.com"}}, "emitted_at": 1671456048322}
+{"stream": "users", "data": {"object": "user", "id": "b14821de-846b-4719-8a94-2453eeafa609", "name": "Iryna Grankova", "avatar_url": null, "type": "person", "person": {"email": "iryna.grankova@airbyte.io"}}, "emitted_at": 1671456048322}
+{"stream": "users", "data": {"object": "user", "id": "1dec424e-8fc2-4265-b82f-731a90dffcd0", "name": "Airbyte", "avatar_url": "https://s3-us-west-2.amazonaws.com/public.notion-static.com/daf02971-a282-49e6-810d-860fcb98ad1d/Screenshot_2021-10-19_at_16.42.29.png", "type": "bot", "bot": {"owner": {"type": "workspace", "workspace": true}, "workspace_name": "Airbyte Testing"}}, "emitted_at": 1671456048322}
```

Comment:
There's PII here - is Karina ok with this?

Reply:
Hi @evantahler
Karina, is this someone from Airbyte?

Reply:
zazmic was a previous company we've worked with - it is OK